axelrod/tests/unit/test_shortmem.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for the Cooperator strategy."""
```

Comment:
Cooperator -> shortmem

Please add a test to catch the array length issue above (array is of length 9 instead of 10 but none of the tests were tripped).